templates/base/base.html

```diff
@@ -8,7 +8,9 @@
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 
-    <title>{ % block tiitle % }{% endblock title}</title>
+    <title>
+      { % block tiitle % }{ % endblock  % }
```

Comment:
ここも同様に
```suggestion
{% block title %}{% endblock %}
```
また、baseディレクトリにbase.htmlを入れるのではなく、templatesディレクトリ直下にbase.htmlでいいと思いますよ。